script.js

```diff
@@ -1,4 +1,469 @@
+  } else if (result === "blackjackplayer") {
+    message += "You drew a Blackjack. You won! 🥳";
+  } else if (result === "blackjackdealer") {
+    message += `The dealer flipped open his card to show <span style="font-size: 20px">${dealerHand[1].name} of ${dealerHand[1].suit}</span>.<br><br> Dealer drew a Blackjack. Tough luck...☹️`;
+  } else {
+    message += `<strong>Hit</strong> to draw a <span style="font-size: 20px">🃏</span> or <strong>Stand</strong> to end your turn.`;
+  }
+  return message;
+};
+
+// Hide & switch button states to indicate active or disabled
+const switchButtonStates = (state) => {
+  let submitButton = document.getElementById("submit-button");
+  let hitButton = document.getElementById("hit-button");
+  let standButton = document.getElementById("stand-button");
+  let restartButton = document.getElementById("restart-button");
+
+  if (state === "player") {
+    submitButton.disabled = true;
+    submitButton.setAttribute("style", 'display: none;')
+    hitButton.disabled = false;
+    hitButton.setAttribute("style", "display: default;");
+    standButton.disabled = false;
+    standButton.setAttribute("style", "display: default;");
+    restartButton.disabled = true;
+    restartButton.setAttribute("style", "display: default;");
+  } else if (state === "end") {
+    submitButton.disabled = true;
+    hitButton.disabled = true;
+    standButton.disabled = true;
+    restartButton.disabled = false;
+    restartButton.setAttribute("style", "display: default;");
+    submitButton.setAttribute("style", "display: none;");
+  } else {
+    submitButton.disabled = false;
+    hitButton.disabled = true;
+    standButton.disabled = true;
+    restartButton.disabled = true;
+    submitButton.setAttribute("style", "display: default;");
+    hitButton.setAttribute("style", "display: none;");
+    standButton.setAttribute("style", "display: none;");
+    restartButton.setAttribute("style", "display: none;");
+  }
+};
+
+// Restart round when restart button is clicked
+const restartRound = () => {
+  playerHand = [];
+  dealerHand = [];
+
+  deck = makeDeck();
+
+  gameMode = "start";
+  playerScore = 0;
+  dealerScore = 0;
+
+  switchButtonStates(gameMode);
+
+  playerState = "safe";
+  dealerState = "safe";
+
+  document.getElementById("image").src =
+    "https://1.bp.blogspot.com/-XnESTN5FkyM/XUx1uHowBfI/AAAAAAAAAKU/9zeKVJuCzZwDsUTgQ2QJe-3SdHlBO1HQQCLcBGAs/s320/tenor.gif";
+
+  restartButton.innerText = "Results";
+  const div = document.getElementById("cards");
+  while (div.hasChildNodes()) {
+    div.removeChild(div.firstChild);
+  }
+  showScore("player");
+};
+
+// ==================== Hit Button helper functions ====================
+
+// Hit function
+const drawCard = () => {
+  if (gameMode === "player") {
+    let card = deck.pop();
+    playerHand.push(card);
+    addCard(card);
+  } else if (gameMode === "dealer") {
+    dealerHand.push(deck.pop());
+  }
+};
+
+// Check if gone bust
+const isBust = (person) => {
```

Comment:
Would not name this function `isBust` since the name implies that it should be true or false but your function does not return a boolean